src/CodeInjector.ts

```diff
@@ -14,13 +14,13 @@ import {
   MAX_SLABS,
   REQUIRED_MODULES,
   VCDAT_VERSION,
-  VCDAT_VERSION_KEY
+  VCDAT_VERSION_KEY,
```

Comment:
Probably dont need the trailing ,

Reply:
When I updated the linting from TSLint (deprecated) to ESLint, I also had to update prettier and other settings. The commas are currently required for some reason. I'll update the linting rules in another PR (this one is already too big as you mentioned)